openmc/mesh.py

```diff
@@ -172,25 +172,26 @@ def _grids(self):
     def vertices(self):
         """Return coordinates of mesh vertices in Cartesian coordinates. Also
            see :meth:`CylindricalMesh.cylindrical_vertices` and
-           :meth:`SphericalMesh.spherical_vertices` for coordinates in other coordinate
-           systems.
+           :meth:`SphericalMesh.spherical_vertices` for coordinates in other
+           coordinate systems.
 
         Returns
         -------
         vertices : numpy.ndarray
             Returns a numpy.ndarray representing the coordinates of the mesh
-            vertices with a shape equal to (ndim, dim1 + 1, ..., dimn + 1).  Can be
-            unpacked along the first dimension with xx, yy, zz = mesh.vertices.
+            vertices with a shape equal to (dim1 + 1, ..., dimn + 1, ndim).  Can
+            be unpacked along the first dimension with xx, yy, zz =
+            np.rollaxis(mesh.vertices, -1).
 
         """
         return self._generate_vertices(*self._grids)
 
     @staticmethod
     def _generate_vertices(i_grid, j_grid, k_grid):
-        """Returns an array with shape (3, i_grid.size+1, j_grid.size+1, k_grid.size+1)
+        """Returns an array with shape (i_grid.size+1, j_grid.size+1, k_grid.size+1, 3)
```

Comment:
```suggestion
        """Returns an array with shape (i_grid.size, j_grid.size, k_grid.size, 3)
```

Reply:
@pshriwise the call to `meshgrid` doesn't add 1 to the size of each array so the docstring is incorrect

Reply:
Thanks! I didn't catch that when reordering these.